tests/testWithFixture.spec.ts

```diff
@@ -0,0 +1,33 @@
+import { test } from './test-options'
+
+test.beforeEach(async ({ page }) => {
+    await page.goto('/')
+})
+
+test('test with fixture', async ({ pageManager, newOwnerFixture }) => {
+
+    const ownerFullName = `${newOwnerFixture[0].firstName} ${newOwnerFixture[0].lastName}`
```

Comment:
In the test, since your fixture returns object instead of array, you don't need to provide the index anymore, so it will be like `newOwnerFixture.firstName`